data/recent.js

```diff
@@ -37,20 +37,20 @@ const highlights = [
 
 const press = [
   {
-    label: 'Washington Post',
-    href: 'https://www.washingtonpost.com/climate-environment/interactive/2023/extreme-heat-wet-bulb-globe-temperature/',
+    label: 'CNN',
+    href: 'https://edition.cnn.com/2024/08/11/climate/olympics-cities-extreme-heat/index.html',
   },
   {
-    label: 'CBS News',
-    href: 'https://www.cbs.com/shows/video/M7H_yUdOiUw_E0fW5l_SP_V786PBhjI0/',
+    label: 'Bloomberg',
+    href: 'https://www.bloomberg.com/news/articles/2024-08-09/clashing-risk-predictions-cast-doubt-on-black-box-climate-models?accessToken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzb3VyY2UiOiJTdWJzY3JpYmVyR2lmdGVkQXJ0aWNsZSIsImlhdCI6MTcyMzIxOTI5MiwiZXhwIjoxNzIzODI0MDkyLCJhcnRpY2xlSWQiOiJTSFlLR0JEV1gyUFMwMCIsImJjb25uZWN0SWQiOiJGMkY1NTFERTc4ODA0RkJBOTk5NEFBMTQ5RTM1NjQ3RCJ9.EbJ-4RSfCLQYocKlLXB1r7Qbo_KKYYNHtobrtIELL_g',
```

Comment:
I think this token is a gift link that only works for [7 days](https://carbonplan.slack.com/archives/C03KVN5L71R/p1723220473910509?thread_ts=1723219867.906879&cid=C03KVN5L71R), so maybe we drop it?

Reply:
Ah using the gift link was a request from Ori 😅 

Looking at the URL, I think the gift link token should just no-op after 7 days. Probably worth cleaning up after that though!